com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/util/BobProjectProperties.java

```diff
@@ -268,6 +271,40 @@ public void remove(String category, String key) {
         return dst;
     }
 
+    static private void mergeMultilineKey(Map<String, Map<String, String>> properties, String group, String keyToMerge) {
```

Comment:
The bob.jar change. We go over all properties of a group (eg "project") in search of properties matching a certain key (eg "dependencies") with a '.' and a number at the end. We collect these and merge them into a comma separated string and set this merged property. The rest of bob works like before and operates on a comma separated list of dependencies.